main.js

```diff
@@ -3,9 +3,36 @@ const EMPTY_HEART = '♡'
 const FULL_HEART = '♥'
 
 // Your JavaScript code goes here!
+const handleServerError = (err) => {
+  const modal = document.querySelector('#modal')
+  modal.querySelector('#modal-message').textContent = err
+  modal.className = ''
+  setTimeout(hideErrorModal,3000)
+}
+
+const hideErrorModal = () => {
+  document.querySelector('#modal').className = 'hidden'
+}
 
+const handleServerSuccess = (res, heartTarget) => {
+  document.querySelector('#modal').className = 'hidden'
+  if(heartTarget.className === 'like-glyph'){
+    heartTarget.textContent = FULL_HEART
+    heartTarget.className = 'activated-heart'
+  }else if(heartTarget.className === 'activated-heart'){
+    heartTarget.textContent = EMPTY_HEART
+    heartTarget.className = 'like-glyph'
+  }
+}
 
+const handleLikeComment = (event) => {
+  const heartTarget = event.target
+  mimicServerCall()
+  .then(res => handleServerSuccess(res, heartTarget))
+  .catch(handleServerError)
+}
```

Comment:
Doesn't look like `res` is being used (and if it were, try naming is `response` next time, just to be more clear). 
```suggestion
const handleServerSuccess = (heartTarget) => {
  document.querySelector('#modal').className = 'hidden'
  if(heartTarget.className === 'like-glyph'){
    heartTarget.textContent = FULL_HEART
    heartTarget.className = 'activated-heart'
  }else if(heartTarget.className === 'activated-heart'){
    heartTarget.textContent = EMPTY_HEART
    heartTarget.className = 'like-glyph'
  }
}

const handleLikeComment = (event) => {
  const heartTarget = event.target
  mimicServerCall()
  .then(handleServerSuccess)
  .catch(handleServerError)
}
```